<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Release</class>
 <widget class="QDialog" name="Release">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="showArea">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>29</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>699</width>
      <height>309</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="chooseBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_Label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>productLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="releaseBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>730</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发布</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="discription_Edit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity_Box">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_Edit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>660</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>标签</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tag_Edit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>640</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productName_Edit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
